README.md

```diff
@@ -4,33 +4,74 @@
 ![security check](https://github.com/gardenlinux/parse_features_lib/actions/workflows/bandit.yml/badge.svg)
 
 # Parse features lib
-This library helps you to work with the gardenlinux/features folder. It parses all info.yamls and builds a tree.
 
-Features (planned):
-* validate CNAMEs
-* validate info.yamls
-* Deduct dependencies from cname_base
+This library includes tooling to build and distribute [Garden Linux](https://github.com/gardenlinux/gardenlinux).
+
+Features:
+
+-   compare APT repositories
+-   parse features
+-   parse flavors
+-   push OCI artifacts to a registry
 
 ## Quickstart
+
+### Example: get a list of features for a given cname
+
 **Inclusion via poetry**:
 
-`parse_features_lib = { git  = "https://github.com/gardenlinux/parse_features_lib", rev="main" }`
+`gardenlinux = { git  = "https://github.com/gardenlinux/python_gardenlinux_lib", rev="0.6.0" }`
+
 ```python
-import parse_features_lib
+from gardenlinux.features import Parser
+
+feature_list = Parser().filter_as_list("aws-gardener_prod")
+for feature in feature_list:
+    print(feature, info["oci_artifacts"])
+```
```

Comment:
The variable `info` is not defined here. Furthermore `features.Parser.filter_as_list()` returns the features included for the given "cname" only.

Reply:
made the example even nicer